pkgs/development/python-modules/terraform-local/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  python-hcl2,
+  packaging,
+  localstack-client,
+}:
+
+buildPythonPackage rec {
+  pname = "terraform_local";
+  version = "0.18.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8opiMd6ZxgWRJIDa0vhZJh5bmsO/CaHgGJ4sdEdxZLc=";
+  };
+
```

```suggestion
  build-system = [ setuptools ];
```
Could you add `setuptools` to `build-system` to avoid a legacy build?

> When unset, the legacy setuptools hooks are used for backwards compatibility.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

https://github.com/NixOS/nixpkgs/pull/323981/commits/2fc795043e1c10eb99dff6e96bf2a7d926c9acf1